### PR TITLE
CA-234 | Add Kotlin collection support for Ktor and SparkJava

### PR DESCRIPTION
**Pull request by sriram-privado**
## Description

Adds support for both SparkJava - https://sparkjava.com/ and Ktor - https://ktor.io/ collection rules. 

This PR depends on the merge of [PR for kotlin collection support](https://github.com/Privado-Inc/privado-core/pull/969).

### CODE REVIEW

**Comment by khemrajrathore** [comment]
@sriram-privado add the privado-core PR link in this PR

File: rules/collections/default/java.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+collections:
+  - id: Collections.SparkJava
+    name: SparkJava Http Framework
+    patterns:
+      - "(?i)Spark.(post|get|delete|put).*"
```
Comment (khemrajrathore) [inline rules/collections/default/java.yaml:5]: @sriram-privado shouldn't we also add the package name for Spark? Or it is Spark?

Comment (sriram-privado) [reply rules/collections/default/java.yaml:5]: `spark.Spark` is the package name. I'll add it here.

**Review by khemrajrathore — APPROVED** [review]
LGTM

**Pull request merged**